### PR TITLE
chore: lazy load ipld formats

### PR DESCRIPTION
**Pull request by fsdiogo**
This needs to merged first https://github.com/ipfs-shipyard/ipld-explorer-components/pull/14 and released.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
     "ipfs-geoip": "^3.0.0",
     "ipfs-redux-bundle": "^5.0.0",
     "ipfs-unixfs": "^0.1.15",
-    "ipld": "^0.18.0",
-    "ipld-explorer-components": "^1.0.1",
+    "ipld": "^0.21.1",
+    "ipld-explorer-components": "^1.1.0",
```
Comment (fsdiogo) [inline package.json]: This needs to be updated to the latest version when released.

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "test:e2e:ci": "cross-env URL=http://localhost:3001 run-p --race test:serve test:e2e",
     "analyze": "source-map-explorer build/static/js/main.*",
     "bundlesize": "bundlesize",
+    "bundle-report": "webpack-bundle-analyzer build/stats.json",
```
Comment (olizilla) [inline package.json]: is this a better bundle analyser than source-map-explorer? do we need both or would you recommend switching over to jus this one?

Comment (fsdiogo) [reply package.json]: I find it easier to look at `webpack-bundle-analyzer`, as it shows more info such as the gzipped size and such (it's possible that `source-map-explorer` does that too, I just don't know about it).

We only need one, I'd switch to `webpack-bundle-analyzer` if that's ok with you!

Comment (olizilla) [reply package.json]: @fsdiogo got for it 👍

**Pull request merged**